### PR TITLE
UI total count bold fix

### PR DESCRIPTION
**Pull request by DeepaliAgrwl**
Please review the ui fix

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/cartproductdisplay/cartproductdisplay.html [hunk 1]
```diff
@@ -46,7 +46,7 @@
             <span class = 'delivery-charges'>160</span>
 			</div>
         </div>
-		 <div>
+		 <div style = "font-weight: bold">
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/cartproductdisplay/cartproductdisplay.html]: @DeepaliAgrwl : we should not be using inlince CSS . Please use CSS heirachy or add a class

**Review by rishabh-behl — CHANGES_REQUESTED** [review]
@DeepaliAgrwl : Pl incorporate comment

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/cartproductdisplay/cartproductdisplay.html (hunk 1 above)
Comment (DeepaliAgrwl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/cartproductdisplay/cartproductdisplay.html]: Comment incorporated

**Review by rishabh-behl — APPROVED** [review]
LGTM

**Pull request merged**